Src/Fare.IntegrationTests/XegerTests.cs

```diff
@@ -121,9 +161,72 @@ public void GeneratedTextIsCorrectWithRexEngine(string pattern)
             @"\Sabc\S{3}111",
             @"^\S\S  (\S)+$",
             @"\\abc\\d",
-            @"\w+1\w{4}",
+            //@"\w+1\w{4}",
```

Comment:
Can you put it back? (Uncomment) We can always tackle the [erratic](http://xunitpatterns.com/Erratic%20Test.html) tests in a separate PR.